package/MDAnalysis/analysis/contacts.py

```diff
@@ -143,9 +143,9 @@
     def is_any_closer(r, r0, dist=2.5):
         return np.any(r < dist)
 
-The first two parameters `r` and `r0` are provided by :class:`Contacts` when it
+The first two parameters ``r`` and ``r0`` are provided by :class:`Contacts` when it
```

Comment:
I started a double backtick rage before realizing that it was not consistent throughout the doc anyway. My reasoning was that `r` and `r0` refer to argument, therefore to code, and that code is in monospace style.